Sources/Public/CalendarView.swift

```diff
@@ -198,60 +198,10 @@ public final class CalendarView: UIView {
 
     guard isReadyForLayout else { return }
 
-    scrollView.performWithoutNotifyingDelegate {
-      scrollMetricsMutator.setUpInitialMetricsIfNeeded()
-      scrollMetricsMutator.updateContentSizePerpendicularToScrollAxis(viewportSize: bounds.size)
-    }
+    _layoutSubviews()
 
-    let anchorLayoutItem: LayoutItem
-    if let scrollToItemContext = scrollToItemContext, !scrollToItemContext.animated {
-      anchorLayoutItem = self.anchorLayoutItem(
-        for: scrollToItemContext,
-        visibleItemsProvider: visibleItemsProvider)
-      // Clear the `scrollToItemContext` once we use it. This could happen over the course of
-      // several layout pass attempts since `isReadyForLayout` might be false initially.
-      self.scrollToItemContext = nil
-    } else if let previousAnchorLayoutItem = self.anchorLayoutItem {
-      anchorLayoutItem = previousAnchorLayoutItem
-    } else {
-      let initialScrollToItemContext = ScrollToItemContext(
-        targetItem: .month(content.monthRange.lowerBound),
-        scrollPosition: .firstFullyVisiblePosition,
-        animated: false)
-      anchorLayoutItem = self.anchorLayoutItem(
-        for: initialScrollToItemContext,
-        visibleItemsProvider: visibleItemsProvider)
-    }
-
-    let currentVisibleItemsDetails = visibleItemsProvider.detailsForVisibleItems(
-      surroundingPreviouslyVisibleLayoutItem: anchorLayoutItem,
-      offset: scrollView.contentOffset)
-    self.anchorLayoutItem = currentVisibleItemsDetails.centermostLayoutItem
-
-    updateVisibleViews(
-      withVisibleItems: currentVisibleItemsDetails.visibleItems,
-      previouslyVisibleItems: visibleItemsDetails?.visibleItems ?? [])
-
-    visibleItemsDetails = currentVisibleItemsDetails
-
-    let minimumScrollOffset = visibleItemsDetails?.contentStartBoundary.map {
-      ($0 - firstLayoutMarginValue).alignedToPixel(forScreenWithScale: scale)
-    }
-    let maximumScrollOffset = visibleItemsDetails?.contentEndBoundary.map {
-      ($0 + lastLayoutMarginValue).alignedToPixel(forScreenWithScale: scale)
-    }
-    scrollView.performWithoutNotifyingDelegate {
-      scrollMetricsMutator.updateScrollBoundaries(
-        minimumScrollOffset: minimumScrollOffset,
-        maximumScrollOffset: maximumScrollOffset)
-    }
-
-    cachedAccessibilityElements = nil
-    if let element = focusedAccessibilityElement as? OffScreenCalendarItemAccessibilityElement {
-      UIAccessibility.post(
-        notification: .screenChanged,
-        argument: visibleViewsForVisibleItems[element.correspondingItem])
-    }
```

Comment:
moved to https://github.com/airbnb/HorizonCalendar/pull/260/files#diff-733b4844a228b4fda6b27e4f88b42a6fc4b04eb589081da658741416a2152bd9R696-R751 with no logic change